src/main/resources/story/web_app/Trello End To End Scenarios.story

```diff
@@ -0,0 +1,90 @@
+Description: Test demoing VIVIDUS capabilities for Web Applications
+
+
+Lifecycle:
+Examples:
+{transformer=FROM_LANDSCAPE}
+|login-button           |By.xpath(//a[@class='Buttonsstyles__Button-sc-1jwidxo-0 kTwZBr'])                            |
+|email                  |sominasvet@gmail.com                                                                         |
```

Comment:
Please, store email and password in a separate file. (Better to store as encoded variables but this is a task #2).